lib/modules/articles-featured-widgets/views/widget.html

```diff
@@ -6,7 +6,8 @@
   <div class="o-container o-container--medium c-article-featured-widget__container">
     <div class="c-article-featured-widget__media">
       <a href="{{ w._article._url }}">
-        {{ image.render(w._article.thumbnail.items[0]._pieces[0]) }}
+        {% set thumbnail = apos.images.first(w._article.thumbnail.items) %}
+        {{ image.render(thumbnail) }}
```

Comment:
Do we need to check if `thumbnail` exists before passing it to `image.render()` or will it check for it inside the macro?

Reply:
I'll fall back to a placeholder image.

Reply:
👍